generator/code_generator.go

```diff
@@ -1932,7 +1943,22 @@ func (m *Message) autoBindFieldToReturnField(field *resolver.Field, autoBindFiel
 		castValue string
 	)
 	if field.RequiredTypeConversion() {
-		fromType := field.SourceType()
+		var fromType *resolver.Type
+		for _, sourceType := range field.SourceTypes() {
+			if typ := m.autoBindSourceType(autoBindField, sourceType); typ != nil {
+				fromType = typ
+				break
+			}
+		}
+		if fromType == nil {
+			panic(
```

Comment:
Was it hard to return an error instead of panicking...?

Reply:
Yes, this function cannot return an error because it is called when evaluating the template.